src/mixins/filing-template-mixin.ts

```diff
@@ -559,129 +567,137 @@ export default class FilingTemplateMixin extends Mixins(DateMixin) {
+    if (filing.changeofRegistration?.parties) {
+      this.setPeopleAndRoles(filing.changeofRegistration.parties)
     }
 
-    // Store share classes
-    if (filing.incorporationApplication) { // *** FUTURE: expand for other filing types
-      if (filing.incorporationApplication.shareStructure) {
-        this.setShareClasses(filing.incorporationApplication.shareStructure.shareClasses)
-      } else {
-        // if it exists, load data from old schema
-        const incorporationApplication = filing.incorporationApplication as any
-        const shareClasses = incorporationApplication.shareClasses as ShareClassIF[]
-        if (shareClasses) {
-          this.setShareClasses(shareClasses)
-        } else {
-          this.setShareClasses([])
-        }
-      }
+    // store Share Classes
+    if (filing.incorporationApplication?.shareStructure) {
+      // load data from new schema
+      this.setShareClasses(filing.incorporationApplication.shareStructure.shareClasses)
+    }
+    if (filing.incorporationApplication?.shareClasses) {
+      // load data from old schema
+      this.setShareClasses(filing.incorporationApplication.shareClasses)
     }
 
-    // Store incorporation agreement type
-    if (filing.incorporationApplication) { // *** FUTURE: expand for other filing types
+    // store Incorporation Agreement Type
+    if (filing.incorporationApplication?.incorporationAgreement) {
       this.setIncorporationAgreementStepData({
-        agreementType: filing.incorporationApplication.incorporationAgreement?.agreementType
+        agreementType: filing.incorporationApplication.incorporationAgreement.agreementType
       })
     }
 
-    // Store certify state
+    // store Certify State
     this.setCertifyState({
       valid: false,
       certifiedBy: filing.header.certifiedBy
     })
 
-    // Store detail comment
+    // store Detail Comment
     // NB: remove the first line (default comment)
     const comment: string = filing.correction.comment || ''
     const detailComment = comment.split('\n').slice(1).join('\n')
     this.setDetailComment(detailComment)
 
-    // Store folio number
+    // store Folio Number
     this.setFolioNumber(filing.header.folioNumber)
 
-    // Store effective date
+    // store Effective Date
     this.setEffectiveDateTimeString(filing.header.effectiveDate)
     this.setIsFutureEffective(filing.header.isFutureEffective)
 
-    // Store Staff Payment data
+    // store Staff Payment
     this.storeStaffPayment(filing)
   }
 
   /**
-   * Parses a draft alteration filing into the store.
+   * Parses a draft Alteration filing into the store.
    * @param filing the alteration filing
    * @param entitySnapshot the latest entity snapshot
    */
   parseAlterationFiling (filing: AlterationFilingIF, entitySnapshot: EntitySnapshotIF): void {
-    // Store business snapshot
-    this.setEntitySnapshot(cloneDeep(entitySnapshot))
+    // store Entity Snapshot
```

Comment:
This is my preferred style for short comments.

For long comments, I prefer properly formatted sentences (with capitalization and punctuation).